### PR TITLE
added Kévin Dunglas as a merger for the Serializer component

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | all |
| Fixed tickets | n/a |

see http://symfony.com/blog/new-symfony-core-team-member-kevin-dunglas

### CODE REVIEW

File: contributing/code/core_team.rst [hunk 1]
```diff
@@ -59,6 +59,9 @@ Active Core Members
     HttpFoundation_, HttpKernel_, Serializer_, Stopwatch_, DoctrineBridge_,
     MonologBridge_, and TwigBridge_ components.
```
Comment (xabbuh) [inline contributing/code/core_team.rst]: This now has to end with a semicolon.

Comment (fabpot) [inline contributing/code/core_team.rst]: fixed

**Comment by stof** [comment]
:+1:

**Comment by weaverryan** [comment]
Congrats @dunglas!

**Pull request merged**